JankStatsSample/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
remove this file if everything's commented?
(and remove from build.gradle if referenced)